Sources/ProtoGen/connector_service.grpc.swift

```diff
@@ -272,7 +272,8 @@ public struct Google_Firebase_Dataconnect_V1alpha_ConnectorServiceNIOClient: Goo
 
 #endif // compiler(>=5.6)
 
-public protocol Google_Firebase_Dataconnect_V1alpha_ConnectorServiceClientInterceptorFactoryProtocol: GRPCSendable {
```

Comment:
This is generated code. I am going to regenerate this with updated protos in a following PR. Will most likely get changed in that PR.

Reply:
Sounds good. Thanks.